recipe/patch_yaml/libgfortran.yaml

```diff
@@ -0,0 +1,7 @@
+if:
+  timestamp_lt: 1743886623000
+  has_depends: libgfortran 5.*
+then:
+  - replace_depends:
+      old: libgfortran 5.*
+      new: libgfortran >=5
```

Comment:
Why not replace this with `libgfortran5 >=<some correct lower bound>`?

Reply:
We want both `libgfortran` to ensure mutual exclusivity.

Reply:
Ok. I don't quite follow any more since it's been too long since I've worked on this bit of the infrastructure.